src/Posts/RepostButton.jsx

```diff
@@ -0,0 +1,170 @@
+const item = props.item;
+
+if (!item) {
+  return "";
+}
+
+const reposts = Social.index("repost", item);
+
+const dataLoading = reposts === null;
+
+const repostsByUsers = Object.fromEntries(
+  (reposts || []).filter((repost) => repost.value.type === "repost").map((repost) => [repost.accountId, repost]),
+);
+
+if (state.hasRepost === true) {
+  repostsByUsers[context.accountId] = {
```

Comment:
This AccountId-to-repost object is being converted to a AccountId-to-String. Am I understanding this correctly? 
However we're only using this object to check if a truthy value exists. If that's true, is this assignment necessary?

Reply:
see updated commit notes about reposts as WIP